temperature/schemas.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class TemperatureBase(BaseModel):
+    date_time: datetime
+    temperature: int
```

Comment:
The `temperature` field is currently defined as an `int`. Consider changing it to `float` to accommodate decimal values, which are common in temperature data.